oracle/statefulset.yaml

```diff
@@ -50,32 +62,14 @@ metadata:
 spec:
   clusterIP: None
   ports:
-    - name: oracle1
+    - name: listener
       protocol: TCP
       port: 1521
       targetPort: 1521
-    - name: oracle2
+    - name: oemexpress
       protocol: TCP
       port: 5500
-      targetPort: 5500      
+      targetPort: 5500
   selector:
     app: oracle-xe
---- 
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: oracle 
---- 
-kind: ClusterRoleBinding
-apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  name: 'poc-dev-oracle-anyuid'
-subjects:
-  - kind: ServiceAccount
-    name: poc-dev
-    namespace: oracle
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: 'system:openshift:scc:anyuid'  
-
+...
```

Comment:
is this needed.

Reply:
No, if you run the commands in the updated README. This makes the demo not dependent on a namespace name.